week1/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	// "errors"
+)
+
+func main() {
+
+	scanner := bufio.NewScanner(os.Stdin)
+	fmt.Print(">")
+	for scanner.Scan(){
+		text := scanner.Text()
+		if text != "" {
+			fmt.Print(process(text))
+			}
+		fmt.Print("\n >")
+	}
+
+}
+
+func process(text string) string{
```

Comment:
function name could be named better, I personally don't expect an output coming out of a `process` function